fe/fe-core/src/main/java/com/starrocks/scheduler/Constants.java

```diff
@@ -8,7 +8,9 @@ public enum TaskType {
         // For Task that can be triggered repeatedly by external.
         MANUAL,
         // For Task that schedules generated by internal periodically.
-        PERIODICAL
+        PERIODICAL,
+        // for Task that is executed asynchronously
+        ASYNC
```

Comment:
This should not use ASYNC, Since user partition does not enable ASYNC and MANUAL for task framework. better to use trigger or event-trigger.Occupy a position first, and then wait for the logic to be sorted out and then hand over the triggered logic to the Task to do it.